castle/cms/indexing/hps.py

```diff
@@ -156,13 +156,6 @@ def gen_audit_query(
     return query
 
 
-def index_in_es(obj):
```

Comment:
see my comments regarding this change on the _social_counts.py and _twitter_monitor.py files